NetCoreRestApi/DataLayer/Repositories/Interfaces/ICategoryRepository.cs

```diff
@@ -0,0 +1,8 @@
+﻿using DataLayer.Models;
+
+namespace DataLayer.Repositories
+{
+    public interface ICategoryRepository<TId> : IRepository<CategoryModel<TId>, TId>
```

Comment:
Fix to ICategoryRepository : IRepository<CategoryModel, Category TId>

Reply:
done